geometry_msgs/PolygonStamped explore_boundary
---
